src/settings.rs

```diff
@@ -111,8 +111,10 @@ impl Settings {
             .map_err(|e| e.into())
     }
 
-    pub fn default_router(&self) -> &KeyedUri {
-        &self.router[&self.update.channel.to_string()]
+    pub fn default_router(&self) -> Option<KeyedUri> {
+        self.router
+            .get(&self.update.channel.to_string())
+            .map(|uri| uri.to_owned())
```

Comment:
this seems like a misnomer since it's not just going to get the uri element of the router keyeduri

Reply:
I've interchangeably used `uri` before to indicate `keyed_uri`. Can't disagree it's a bit ambiguous. We'll have to take a pass at clearing this up.

Reply:
and fixed in 1da305351aa112d15c788078ccea559235500813